docs/crds/avinfrasetting.md

```diff
@@ -167,3 +168,12 @@ For the subset of ingresses, that refer to an ingress class which in turn refers
 For passthrough routes/ingresses, setting `l7Settings:shardSize` present in AviInfrasetting CRD overrides setting `L7Settings.passthroughShardSize` present in values.yaml. <br>
 **Note**:  Value `DEDICATED` is not supported when AviInfrasetting CRD is applied to the passthrough route/ingress.
 
+#### Configure IPv6
+
+AviInfraSetting CRD can be used to enable IPv6, IPv4 or both IPv4 and IPv6 vips on virtualservices created by AKO. 
```

Comment:
Please mention that this feature is in techpreview stage in this release

Reply:
added tech preview to title